lib/kamal/commands/builder/pack.rb

```diff
@@ -0,0 +1,26 @@
+class Kamal::Commands::Builder::Pack < Kamal::Commands::Builder::Base
+  def push
+    combine \
+      pack(:build,
+        config.repository,
+        "--platform", platform,
+        "--builder", pack_builder,
+        buildpacks,
+        "-t", config.absolute_image,
+        "-t", config.latest_image,
+        "--env", "BP_IMAGE_LABELS=service=#{config.service}",
+        *argumentize("--env", secrets, sensitive: true),
```

Comment:
Is using environment variables the standard way to get secrets into a buildpack?

Reply:
@djmb Yes, they only have the `--env` flag. 

I just tested building with a few secrets because I was concerned they'd end up in the final image but they don't. 

I just found this in the docs site though. TLDR; It's just a build-time env var, they're not available at image runtime. So they're naturally "secret", neat.

https://buildpacks.io/docs/for-platform-operators/how-to/integrate-ci/pack/cli/pack_build/#options
```
  -e, --env stringArray              Build-time environment variable, in the form 'VAR=VALUE' or 'VAR'.
                                     When using latter value-less form, value will be taken from current
                                       environment at the time this command is executed.
                                     This flag may be specified multiple times and will override
                                       individual values defined by --env-file.
                                     Repeat for each env in order (comma-separated lists not accepted)
                                     NOTE: These are NOT available at image runtime.
```